src/common-crypto/CryptoBackend.ts

```diff
@@ -113,6 +115,16 @@ export interface SyncCryptoCallbacks {
      */
     processKeyCounts(oneTimeKeysCounts?: Record<string, number>, unusedFallbackKeys?: string[]): Promise<void>;
 
+    /**
+     * Handle the notification from /sync or /keys/changes that device lists have
+     * been changed.
+     *
+     * @param syncData - Object containing sync tokens associated with this sync
+     * @param deviceLists - device_lists field from /sync, or response from
+     * /keys/changes
+     */
+    processDeviceLists(syncData: ISyncStateData, deviceLists?: Required<ISyncResponse>["device_lists"]): Promise<void>;
```

Comment:
can we define the type of `deviceLists` from first principles rather than by reference to `ISyncResponse`? It's an `IDeviceLists`, which is much simpler.

Reply:
Can we use instead the `IDeviceLists` definition in the `sync-accumulator.ts` ? (Where from `IToDeviceEvent` definition is also coming)

Reply:
I'm afraid I don't understand the question. Yes, `IDeviceLists` is defined in `sync-accumulator.ts`, and we should import it here.

Reply:
I misunderstood your first comment, I repeated the same idea :)